### DISCUSSION TITLE
Magic sign-in link via email

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3068] [Feature Requests]
Ability to login seamlessly via a OTP link delivered to the user's email address.

**Comment by adanielyan** [comment]
Has there been any work done on this?

**Reply by rijkvanzanten** [reply]
It's an open feature request, there hasn't been any work done on it yet. The first step into making this go somewhere real is having a proper RFC for it. Wanna help out with that @adanielyan? :) It's basically filling out https://github.com/directus/directus/discussions/new?category=feature-requests in as much detail as we can 🙂  (I'll copy the response into this thread so we don't lose the upvotes 👍🏻 )

**Reply by adanielyan** [reply]
I would be more than happy to do that, but I am not that familiar with the directus codebase to answer some of the questions on the form you shared, such as Drawbacks or Adoption strategy.

**Reply by rijkvanzanten** [reply]
That's fine! I can help fill in the blanks :) The most important bit to me is the requirements list and technical implementation. For example, what link does it generate, what type of code is in the link, how long is it valid, is there a way to customize where it redirects to after login (similar to SSO) etc 🙂

**Reply by adanielyan** [reply]
https://github.com/directus/directus/discussions/17888

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by rijkvanzanten** [comment]
Closing this in favor of https://github.com/directus/directus/discussions/3068, which is a more detailed in-depth RFC around the same feature request (and it also has the requirement amount of upvotes!)
